SKIRT/core/SimulationItemRegistry.cpp

```diff
@@ -546,6 +547,7 @@ SimulationItemRegistry::SimulationItemRegistry(string version, string format)
     ItemRegistry::add<FullInstrument>();
     ItemRegistry::add<AllSkyInstrument>();
     ItemRegistry::add<PerspectiveInstrument>();
+    ItemRegistry::add<HEALPixSkyInstrument>();
```

Comment:
I suggest placing the new instrument one line up (i.e. after AllSky and before Perspective). This determines the order in which the instruments are shown in a console Q&A session or in MakeUp.